src/test/java/restel/core/model/RestelTestWrapperTest.java

```diff
@@ -0,0 +1,31 @@
+package restel.core.model;
+
+import com.techconative.restel.core.model.RestelTestApiDefinition;
+import com.techconative.restel.core.model.RestelTestApiWrapper;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RestelTestWrapperTest {
+  @Test
+  public void testRestelTestMethod() {
+    RestelTestApiWrapper restelTestWrapper = new RestelTestApiWrapper();
+
+    restelTestWrapper.setTestApiDefinition(new RestelTestApiDefinition());
+    Assert.assertEquals(new RestelTestApiDefinition(), restelTestWrapper.getTestApiDefinition());
+
+    restelTestWrapper.setTestApiWrapperName("testApiWrapperName");
+    Assert.assertEquals("testApiWrapperName", restelTestWrapper.getApiUniqueName());
+
+    restelTestWrapper.setTestApiWrapperDescription("a read test that returns 500 status code");
```

Comment:
Don't see a much of value in setting and getting from the same object. For ex, we can set it in `RestelTestApiDefinition` instance and assert it from the wrapper.

Reply:
For the second and the third assertions, we set via wrapper's setters and then access them through overridden getters from the Restel API interface. 

Though to test this since we parse and create the test sheets and use them via the interface.

In case of wrappers, the API name and description from `RestelTestApiDefinition` is not used.

Reply:
Created a full RestelTestDefinition instance with values and read via the wrapper getters